test/e2e/specs/site-editor/site-editor-inserter.spec.js

```diff
@@ -64,4 +62,396 @@ test.describe( 'Site Editor Inserter', () => {
 
 		await expect( blockLibrary ).toBeHidden();
 	} );
+
+	test( 'should open the inserter to patterns tab if using zoom out and stay in zoom out when closing the inserter', async ( {
+		InserterUtils,
+	} ) => {
+		const zoomOutButton = InserterUtils.getZoomOutButton();
+		const inserterButton = InserterUtils.getInserterButton();
+		const blockLibrary = InserterUtils.getBlockLibrary();
+
+		await zoomOutButton.click();
+		await expect( await InserterUtils.getZoomCanvas() ).toBeVisible();
```

Comment:
```suggestion
		await expect( InserterUtils.getZoomCanvas() ).toBeVisible();
```

I think there's no need to `await` canvas locator. Usually, you would write it like this - `await expect( page.locator( '.is-zoomed-out' ) ).toBeVisible();`.

Linter will remove extra await if you try to put it there.